src/tabcenter.js

```diff
@@ -92,9 +104,15 @@ TabCenter.prototype = {
     }
   },
   async showNewTabMenu() {
-    this._newTabMenuShown = true;
+    if (!browser.contextualIdentities) {
+      return; // no support
+    }
     // Create the identities
     const identities = await browser.contextualIdentities.query({});
+    if (!identities) {
```

Comment:
In Firefox 55 Beta always, and Nightly 56 with Enable Container Tab option is disabled in Options.